ci_framework/roles/ci_network/molecule/default/converge.yml

```diff
@@ -0,0 +1,35 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Converge
+  hosts: all
+  roles:
+    - role: "ci_network"
+  tasks:
+    - name: Get nmcli connections
+      register: nmcli_existing_connections
+      ansible.builtin.command:
+        cmd: nmcli -t -f NAME,DEVICE con show
+
+    - name: Debug existing connections
+      ansible.builtin.debug:
+        var: nmcli_existing_connections.stdout_lines
+
+    - name: Ensure we have the expected vlan20
+      ansible.builtin.assert:
+        that:
+          - "'molecule-test:eth0.20' in nmcli_existing_connections.stdout_lines"
```

Comment:
Should we ensure is UP? Should we check if the VLAN id is the expected one and that the IP too?

Reply:
added a second assertion block based on the interface config output from nmcli (as a second call)